src/components/common/Stepper.vue

```diff
@@ -1,7 +1,13 @@
 <template>
-  <v-container id="step-buttons-container">
+  <v-card id="step-buttons-container">
```

Comment:
How is v-card more appropriate than v-container? (I don't disagree, just curious.)

https://vuetifyjs.com/en/components/cards/
https://vuetifyjs.com/en/components/grids/#v-container